jbpm-wb-case-mgmt/jbpm-wb-case-mgmt-api/src/main/java/org/jbpm/workbench/cm/service/CaseManagementService.java

```diff
@@ -81,4 +81,6 @@
 
     List<ProcessDefinitionSummary> getProcessDefinitions(String containerId);
 
+    List<CaseCommentSummary> getComments(String serverTemplateId, String containerId, String caseId, int currentPage, int pageSize);
```

Comment:
Could we just add the new parameter to the other `getComments` method? Seems like we don't need to keep both.